qt-ozwdaemon/mqttpublisher.cpp

```diff
@@ -16,7 +16,11 @@ mqttpublisher::mqttpublisher(QSettings *settings, QObject *parent) : QObject(par
     this->m_client = new QMqttClient(this);
     this->m_client->setHostname(settings->value("MQTTServer", "127.0.0.1").toString());
     this->m_client->setPort(static_cast<quint16>(settings->value("MQTTPort", 1883).toInt()));
-
+    this->m_client->setClientId("qt-openzwave");
+    if (settings->contains("MQTTUsername") && settings->contains("MQTTPassword")) {
+        this->m_client->setUsername(settings->value("MQTTUsername", "").toString());
+	this->m_client->setPassword(settings->value("MQTTPassword", "").toString());
```

Comment:
indentation?

Reply:
This is something that frustrates me.  I used VS Code and copied the indentation from another line.  Why does it appear different on GitHub! :)

Reply:
Line 22 starts with a tab character, line 21 with 8 spaces, it should probably be spaces to be consistent with the rest of the file.

Reply:
Thanks.  Fixed :)